documentation/sphinx/source/release-notes/release-notes-730.rst

```diff
@@ -2,6 +2,20 @@
 Release Notes
 #############
 
+7.3.17
+======
+* Same as 7.3.16 release with AVX enabled.
+
+7.3.16
+======
+* Released with AVX disabled.
```

Comment:
many of these are syntactically incorrect. there is inconsistency in the use of `<,>` characters and `(,)` Please fix these.